docs/user/firststeps.rst

```diff
@@ -30,137 +27,130 @@ Our first Renga project
 
 First, create a project directory:
 
-.. code-block:: console
-
-    $ mkdir -p ~/renga-projects/test-project
-    $ cd ~/renga-projects/test-project
-
-Set up your platform credentials (using the demo ``docker-compose``
-configuration, enter ``demo/demo`` for username/password:
-
-.. code-block:: console
-
-    $ renga login http://localhost
-    Username:
-    Password:
-    Access token has been stored in: ~/Library/Application Support/Renga/config.yml
-
-Register the project with the platform (note that the ``autosync`` option
-is mandatory for now):
-
-.. code-block:: console
-
-    $ renga init --autosync
-
-Creating a project deployment
------------------------------
-
-Renga can be used to launch computational tasks that run inside docker
-containers. For this, you must first create an execution "context" which
-defines the task to be carried out. This is then followed by executing the
-context on one of the platform deployment engines.
-
-Create an execution context using a docker container:
-
-.. code-block:: console
-
-    $ renga contexts create hello-world
-    14971456
-
-This creates a context - we can now run it on an engine:
-
-.. code-block:: console
-
-    $ renga contexts run 14971456 docker
-    16d1491b
-
-To see all of your existing contexts:
+::
 
-.. code-block:: console
+    $ mkdir -p ~/renga-projects/myproject
+    $ cd ~/renga-projects/myproject
 
-    $ renga contexts list
-    ID               VERTEX_ID    IMAGE
-    ---------------  -----------  -----------
-    16d1491b         4168         hello-world
+Initialize the project as a Renga project:
```

Comment:
IMHO here we should point to `renga-python` documentation and not repeat ourself

Reply:
Right, but I think we do want some sort of "First steps" other than a link in the main repo as well?

Reply:
also, the "first steps" in the main renga repo should soon contain instructions that include the UI etc. and we want all of this to be on the same page so the user doesn't have to click around too much